config/config.go

```diff
@@ -0,0 +1,113 @@
+package config
+
+import (
+	"io/ioutil"
+	"os"
+	"runtime"
+	"strings"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/pengsrc/go-shared/check"
+)
+
+// A Config stores a configuration of BeyondFTP.
+type Config struct {
+	Service struct {
```

Comment:
Please use the `go-storage` connection string instead. Maybe we need an RFC to define the config format.

(I prefer toml for now.)